Taiga story700 tasks

mareknovotny opened the pull request:
https://tree.taiga.io/project/rdruss-jboss-migration-windup-v3/task/701
https://tree.taiga.io/project/rdruss-jboss-migration-windup-v3/task/703
https://tree.taiga.io/project/rdruss-jboss-migration-windup-v3/task/707
https://tree.taiga.io/project/rdruss-jboss-migration-windup-v3/task/706

klinki commented:
> A few issues:
> 
> - The local rules should be available in a collapsed panel, like in the mockups
> 
I think this one is part of 708 - Hide other advanced options in a collapsible panel https://tree.taiga.io/project/rdruss-jboss-migration-windup-v3/task/708

mareknovotny commented:
@klinki no, it is https://tree.taiga.io/project/rdruss-jboss-migration-windup-v3/task/706 ;)

jsight commented:
@mareknovotny - Yeah, that is what I was thinking.

Also, feel free to defer the back/next button complaint that I had. I can add a new task for that to the end of the sprint as this isn't the only step that doesn't have it working. :)

mareknovotny commented:
@jsight that is fine, I will fix it in this PR too

mareknovotny commented:
@jsight actually I talked with @klinki about Back button routing and it seems my changes should be in line with other back buttons in wizard and in Config Analysis outside wizard, so I won't change it in this user story. We need to create additional task for this I think.

jsight commented:
@mareknovotny - I agree. I hadn't realized that when I initially commented. New task is here:
https://tree.taiga.io/project/rdruss-jboss-migration-windup-v3/task/777?kanban-status=543760

mareknovotny commented:
retest

mareknovotny commented:
retest

mareknovotny commented:
So we have discussed on mtg some possibilities how to enable multiple selections for Transformation paths. Right now single selection is working fine. The problem is to have a single checkbox for Cloud path with multiple targets (we have openshift and cloud-readiness targets) to be added into radio button selected Transformation path.

jsight commented:
@mareknovotny - A couple of notes:

- The padding on the selections for the transformation path seem a bit off to me. Can you increase this to have spacing closer to the mockup?
- The wizard bar at the top still says "Configure Analysis" instead of "Configure the Analysis"
- It isn't on the mockups (because the cloud-readiness checkbox isn't there), but could you also change the hint text? I think the verbiage that is there now can go under the select buttons. Then add a new line under the cloud checkbox: "Check to enable rules for preparing your application for deployment on a cloud service"

mareknovotny commented:
@jsight sure, I will change that, it was just first POV to get it work

mareknovotny commented:
I have locally test failures on non related tests, so trying to check if it is only on my local or not, rebased too

mareknovotny commented:
I figured finally what's wrong with tests and why AnalysisContext doesn't have propagated flag even on UI is shown correctly. (cloning of AnalysisContext needs manual additions :( )

mareknovotny commented:
@jsight wrt your notes from https://github.com/windup/windup-web/pull/323#issuecomment-295781408:
* I added some padding and margin to radio buttons labels
* the title is there already correct, I don't know why you should see it without "the", see the first commit on this PR -> https://github.com/windup/windup-web/pull/323/commits/daabac52aa027860b4aee0df26e36f579bf03590
* I moved the helper text to radio buttons and added new one for just checkbox with cloud readiness flag

mareknovotny commented:
@klinki look now, not ideal, but better ;)

mareknovotny commented:
removed required asterisk from INcluded Packages as that is not mandatory.
Some help text changes will follow as Marc requested it on showcase of the form
